deploy/src/native_to_erc/foreign.js

```diff
@@ -25,7 +26,10 @@ const {
   FOREIGN_REQUIRED_BLOCK_CONFIRMATIONS,
   BRIDGEABLE_TOKEN_NAME,
   BRIDGEABLE_TOKEN_SYMBOL,
-  BRIDGEABLE_TOKEN_DECIMALS
+  BRIDGEABLE_TOKEN_DECIMALS,
+  DEPLOY_REWARDABLE_TOKEN,
```

Comment:
we need to modify `src/loadEnv.js` in order to validate all defined parameters there.

Reply:
Done: This is the address of [`ValidatorSet` contract](https://github.com/poanetwork/pos-contracts/blob/master/contracts/ValidatorSetAuRa.sol) which contains network validators. Ok, I've renamed the param: https://github.com/poanetwork/poa-bridge-contracts/pull/129/commits/47f766b6c90759281abffee38e0c0834417ed727.